packages/orbit-design-tokens/src/dictionary/definitions/global/elevations.json

```diff
@@ -186,6 +300,44 @@
         }
       },
       "raised-reverse": {
+        "background": {
+          "type": "color",
+          "category": "elevation",
+          "value": "{foundation.palette.white.normal}",
+          "deprecated": true,
+          "deprecated-replace": "{foundation.palette.white.normal}",
+          "deprecated-version": "9.0.0"
```

Comment:
Following my previous comment…

I see this is being deprecated with the replacement pointing to `palette.white.normal`. But then the `level3-reverse` background color is also being created, with the value of `palette.white.normal`.

So I think we are not going through the correct approach. We either:
- Use `level3-reverse` as a replacement here and use `palette.white.normal` as a value there

**OR**

- Use `palette.white.normal` as a replacement here but do not define `level3-reverse` at all

I personally prefer the second option, as it will remove the overdefinition of tokens entirely. But I am open for discussing (and accepting) the first option as well

Reply:
Ok, gotcha, I'll remove the background tokens and adapt things accordingly. For some reason, I thought we wanted to keep backgrounds as well but I see they are barely used, so there's no point.